lua/nvim-gps/init.lua

```diff
@@ -5,60 +5,63 @@ local gps_utils = require("nvim-gps.utils")
 
 local M = {}
 
-local config = {
+-- Default configuration that can be overridden by users
+local default_config = {
+	enabled = true,
 	icons = {
 		["class-name"] = ' ',
 		["function-name"] = ' ',
 		["method-name"] = ' ',
 		["container-name"] = '⛶ ',
 		["tag-name"] = '炙'
 	},
-	languages = {
-		["bash"] = true,       -- bash and zsh
-		["c"] = true,
-		["cpp"] = true,
-		["elixir"] = true,
-		["fennel"] = true,
-		["go"] = true,
-		["html"] = true,
-		["java"] = true,
-		["jsx"] = true,
-		["javascript"] = true,
-		["lua"] = true,
-		["ocaml"] = true,
-		["python"] = true,
-		["ruby"] = true,
-		["rust"] = true,
-		["tsx"] = true,
-		["typescript"] = true,
-	},
 	separator = ' > ',
 }
 
+-- Languages specific default configuration must be added to configs
```

Comment:
@SmiteshP, what do you think about this solution?

Reply:
Works great!